copter/source/docs/traditional-helicopter-swashplate-setup.rst

```diff
@@ -155,9 +155,45 @@ that produces zero thrust. In Copter 3.6 and ealier, the minimum collective pitc
 could command was done with the parameter ``H_COL_LAND_MIN``. 
 
 Set :ref:`H_SV_MAN <H_SV_MAN>` to 3 to have the swashplate move to the mid position.  Adjust :ref:`H_COL_MID <H_COL_MID>` until the blade
-pitch angle measures the desired collective blade pitch that corresponds to zero thrust.  
+pitch angle measures the desired collective blade pitch that corresponds to zero thrust.
 
+.. note:: If the minimum or maximum collective is measured in ACRO (see below for impact of ACRO on collective pitch curve vs stabilized modes), the value will be less than those since some servo movement is reserved for cyclic. Of course, any swashplate tilt due to pilot or autopilot input will interefere with the measurements also.
 
+ACRO versus STABILIZED Collective Curves
+========================================
+
+ArduPilot provides a programmable curve for stabilized modes (all except ACRO). This curve is set by the ``IM_STB_COL_x`` parameters as shown in the graph below. Only ACRO mode uses the collective extremes that were setup above.
+
+.. image:: ../images/collective_curves.png
+   :target: ../_images/collective_curves.png
+
+By default, these settings mirror the ACRO mode curve, and are satisfactory if the -2° to +10° range is setup. This will give the following values:
+
+==============      =====
+Throttle Stick      Pitch
+==============      =====
+Low                 -2
+Mid                 +4 (close to hover)
```

Comment:
To reduce confusion with the collective mid (H_COL_MID), I would suggest you replace mid with center in your tables

Reply:
will do